tools/picard_collectaggregationmetrics.cwl

```diff
@@ -0,0 +1,36 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: picard_collectaggregationmetrics
```

Comment:
Is this tool supposed to be in this repo?!

Reply:
Sneaky cwl found its way into the repo

Reply:
Deleted